zerver/lib/i18n.py

```diff
@@ -76,6 +76,12 @@ def get_available_language_codes() -> List[str]:
     return codes
 
 
+def get_available_language_locales() -> List[str]:
+    language_list = get_language_list()
+    locales = [language["locale"] for language in language_list]
```

Comment:
Should we be getting language["code"] here instead of language["locale"]?

![Screenshot from 2021-06-10 00-31-18](https://user-images.githubusercontent.com/7190633/121413335-34488a00-c983-11eb-9904-01fbcb4c1728.png)